cfgov/v1/migrations/0008_filter_controls_move_page_type.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import json
+from django.db import migrations, models
+import v1.atomic_elements.atoms
+import wagtail.wagtailcore.fields
+import wagtail.wagtailcore.blocks
+import wagtail.wagtailimages.blocks
+
+
+def move_page_type(apps, schema_editor):
+    Revision = apps.get_model('wagtailcore.PageRevision')
+    for model_name in ['SublandingFilterablePage', 'BrowseFilterablePage']:
+        model = apps.get_model('v1.%s' % model_name)
+        for page in model.objects.all():
+            update_page(page)
+            revisions = get_revisions_to_update(page, Revision)
+            for revision in revisions.values():
+                update_revision(revision)
+
+def update_page(page):
+    for i, block in enumerate(page.content.stream_data):
```

Comment:
doesn't look like `i` is used, which means you could just iterate `for block in ...`